lib/radiator/contribution/person.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Radiator.Contribution.Person do
+  @moduledoc """
+  TODO: Does this all fit into one context or should it be
+        separate? People can stand on their own, without
+        being used for contributions.
+        AudioContribution could fit into AudioMeta.
+
+        There are also different kinds of contributions:
+
+        - podcast contributor
+        - episode contributor
+        - audio contributor
+
+        Can we get away with audio + podcast?
+        Or do we need episode as well?
+
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+  import Arc.Ecto.Changeset
+
+  import Ecto.Query, warn: false
+
+  alias Radiator.Auth.User
+  alias Radiator.Media
+
+  schema "people" do
+    field :real_name, :string
+    field :display_name, :string
+    field :nick_name, :string
+    field :gender, :string
+    field :email, :string
+
+    field :avatar, Media.PersonAvatar.Type
+
+    belongs_to :user, User
+
+    timestamps()
```

Comment:
Good questions. Network-global maybe, which means they need a network reference. And can only be edited by network owners (except each user who can update his/her own data).

Interesting thought as well about multiple people representing the same entity. My gut feeling is that we want this, but maybe not necessarily right now, to prioritize more urgent features?

Reply:
A person could also be in more than one network so that a person can be a contributor in podcasts from different networks and the person can be identified over multiple networks.

Reply:
But then who has permissions to edit that person? Just for this we would need to create an entity above networks so we can manage permissions for it. I feel like this creates bigger problems than it solves.

Reply:
Thats a good question.

In the best scenario i would say that contributors can edit their own data and we have some kind of unique id for every contributor that can be used so that contributors can be tracked over multiple networks and radiator installations. But thats even farer off for now.

So probably you are right, that currently the best choice is to connect a person (contributor) to one network.